### PR TITLE
add ccache/4.5.1

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **lib/1.0**

closes https://github.com/conan-io/conan-center-index/issues/8584

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
```
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_sse2_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_sse41_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_avx2_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_avx512_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_sse2_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_sse41_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_avx2_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
/Library/Developer/CommandLineTools/usr/bin/ranlib: archive member: ../../../../lib/libblake3.a(blake3_avx512_x86-64_unix.S.o) cputype (16777223) does not match previous archive members cputype (16777228) (all members must match)
```

What? Can't reproduce this issue.

File: recipes/ccache/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```
Comment (jgsogo) [inline recipes/ccache/all/CMakeLists.txt:5]:
```suggestion
conan_basic_setup(TARGETS)
```

**Review by jgsogo — APPROVED** [review]

File: recipes/ccache/all/CMakeLists.txt (hunk 1 above)
Comment (SpaceIm) [reply recipes/ccache/all/CMakeLists.txt:5]: It doesn't really matter in cmake wrapper of the recipe, why do you want TARGETS?

**Review by uilianries — APPROVED** [review]
LGTM

**Review by madebr — APPROVED** [review]

**Pull request merged**